mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/routes/v1/AvtaleRoutes.kt

```diff
@@ -29,6 +31,25 @@ fun Route.avtaleRoutes() {
     val excelService: ExcelService by inject()
 
     route("/api/v1/internal/avtaler") {
+        authenticate(AuthProvider.AZURE_AD_AVTALER_SKRIV.name, strategy = AuthenticationStrategy.Required) {
```

Comment:
Kan være kjekt å få implementert noen tester for dette. Kan evt. se på det sammen :)

Reply:
Yes, kan ta en felles økt på testing av routes.